index.html

```diff
@@ -228,26 +235,34 @@ <h3><img src="Resources/Images/Rectangle57.png" alt="Skills"> <span class="about
                 <li><a href="#">Terminal</a></li>
             </ul>
         </div>
-    </section>
-    <section class="contact">
-        <h2>I'm always <br>interested in hearing <br>about new projects, <br>so if you'd like to <br>chat please get in <br>touch.</h2>
-        <form action="https://formspree.io/f/xrgjovwd" method="post" id="f1">
-            <input name="text" id="text1" type="text" maxlength="30" placeholder="Full name" required>
-            <input name="email" id="email1" type="email" placeholder="Email adress" required>
-            <textarea name="message" id="text_zone1" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
-            <input type="submit" value="Get in touch" >
-        </form>
-        
-    </section>
-    <div class="footericon">
-      <ul>
-        <li><a href="#"><img src="Resources/Images/Vector.png" alt="Github"></a></li>
-        <li><a href="#"><img src="Resources/Images/Group0.png" alt="Linkin"></a></li>
-        <li><a href="#"><img src="Resources/Images/Vector5.png" alt="Angelist"></a></li>
-        <li><a href="#"><img src="Resources/Images/Vecto2r.png" alt="Twitter"></a></li>
-        <li><a href="#"><img src="Resources/Images/Vector13.png" alt="Medium"></a></li>
-      </ul>
-      <div class="line1"></div>
-    </div>
+        </footer>
+        <section id="contact">
+            <h2>I'm always <br>interested in hearing <br>about new projects, <br>so if you'd like to <br>chat please get in <br>touch.</h2>
+            <form action="https://formspree.io/f/xqknwlkr" method="post" id="f1">
+                <input name="text" id="text1" type="text" maxlength="30" placeholder="Full name" required>
+                <input name="email" id="email1" type="email" placeholder="Email adress" required>
+                <textarea name="message" id="text_zone1" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+                <input type="submit" value="Get in touch" >
+            </form>
+            <form action="https://formspree.io/f/xqknwlkr" method="post" id="f2">
+                <span>
+                    <input name="text" id="text2" type="text" maxlength="30" placeholder="First name" required>
+                    <input name="text" id="text" type="text" maxlength="30" placeholder="Second name" required>
```

Comment:
- [ ] Kindly change the input color of your form input to match the Figma design
![image](https://user-images.githubusercontent.com/46839548/187426718-78f81cfc-31ef-4daf-b63f-2c5d155a9ba1.png)